de-lib/src/main/java/org/iplantc/de/theme/base/client/collaborators/GroupDisplayStrings.properties

```diff
@@ -1,3 +1,5 @@
+addGroup=Add List
+deleteGroup=Delete List
 deleteGroupConfirm=By deleting the collaborator list {0}, any data that has been shared with this list will immediately be un-shared.  Do you wish to continue deleting the collaborator list?
```

Comment:
My thinking on the distinction between collaborator lists and teams was merely one of visibility. Users can see when they're in a team and teams can have multiple administrators. Other than that, the functionality was quite similar to that of collaborator lists.